04-kogito-visas/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Kogito Visas</title>
+
+    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+</head>
+<body>
+<div class="container" role="main">
+    <div class="jumbotron">
+        <h1>Kogito Visas</h1>
+
+        <p>Visa Applications approval service</p>
+    </div>
+    <div class="page-header">
+        <div class="btn-group pull-right" role="group" style="padding-top: 10px;">
+            <button class="btn btn-primary" onclick="load();">
+                <span class="glyphicon glyphicon-refresh"></span> Refresh
+            </button>
+        </div>
+        <h1>Visa applications</h1>
+    </div>
+    <div class="row">
+        <div class="col-md-12">
+            <table class="table">
+                <thead>
+                <tr>
+                    <th>Traveller name</th>
+                    <th>Destination</th>
+                    <th>Duration</th>
+                    <th>Passport number</th>
+                    <th>Nationality</th>
+                    <th></th>
+                </tr>
+                </thead>
+                <tbody id="content">
+                <!-- filled using Ajax -->
+                </tbody>
+            </table>
+        </div>
+    </div>
+</div>
+</div>
+<script>
+    var graphql = "http://localhost:8180/graphql";
```

Comment:
do we have to include the `http://localhost:8180` here? wouldn't it be enough to use `/graphql` only?

Reply:
its not running on the same port. ideally, this should be configured somewhere. any suggestion?